patches/gtk+-3/0001-Initial-Managarm-support.patch

```diff
@@ -0,0 +1,78 @@
+From fc3ab7aba6b4f70b3867cf83113c61dd4817d444 Mon Sep 17 00:00:00 2001
+From: Dennis Bonke <admin@dennisbonke.com>
+Date: Sun, 3 Apr 2022 22:48:43 +0200
+Subject: [PATCH] Initial Managarm support
+
+Signed-off-by: Dennis Bonke <admin@dennisbonke.com>
+---
+ gtk/a11y/gtkaccessibility.c |  4 ++++
+ gtk/gtkutils.c              | 10 ++++++++--
+ meson.build                 |  2 +-
+ 3 files changed, 13 insertions(+), 3 deletions(-)
+
+diff --git a/gtk/a11y/gtkaccessibility.c b/gtk/a11y/gtkaccessibility.c
+index 7f0e5207d9..7231eb3fb6 100644
+--- a/gtk/a11y/gtkaccessibility.c
++++ b/gtk/a11y/gtkaccessibility.c
+@@ -974,6 +974,9 @@ do_window_event_initialization (void)
+ void
+ _gtk_accessibility_init (void)
+ {
++#ifdef __managarm__
++	return;
++#else
```

Comment:
Why do we patch this out?

Reply:
So we don't have to use dbus, and get a 30 second timeout delay in gtk startup. We lose no functionability from what I could see.

Reply:
Add a comment to indicate that this works around a missing DBus port.

Reply:
Not missing, rather broken. I’ll add a comment noting that we’re working around broken DBus here.